src/ios/UsabillaCordova.swift

```diff
@@ -98,8 +104,37 @@ import Usabilla
             self.success(completed: true)
         }
     }
+
+    // Set Dismiss to close the campaign
+    @objc(dismiss:)
+    func dismiss(_ command: CDVInvokedUrlCommand) {
+        self.command = command;
+        let _ = Usabilla.dismiss()
+        self.success(completed: true)
+    }
+
+    // Default Data masking
+    @objc(getDefaultDataMasks:)
+    func getDefaultDataMasks(_ command: CDVInvokedUrlCommand) {
+        self.command = command;
+        let str = Usabilla.defaultDataMasks
+        self.success(completed: str)
+    }
+
+    // Set Data masking
+    @objc(setDataMasking:)
+    func setDataMasking(_ command: CDVInvokedUrlCommand) {
+        self.command = command;
+        self.extractCustomVariables(command: command);
+        if let maskCharacter = self.maskChar?.first, let mask = self.masks {
+            Usabilla.setDataMasking(masks: mask, maskCharacter: maskCharacter)
+        } else {
+            Usabilla.setDataMasking(masks: Usabilla.defaultDataMasks, maskCharacter: "X")
```

Comment:
Isn't the `X` also present in Usabilla as constant for `default masking character` or something similar?

Reply:
Once we have all constant in a place we can have it right now i didnt find anything like default masking character in sdk.